model/websocket_message.go

```diff
@@ -44,6 +44,8 @@ const (
 	WEBSOCKET_EVENT_CHANNEL_VIEWED      = "channel_viewed"
 	WEBSOCKET_EVENT_PLUGIN_ACTIVATED    = "plugin_activated"   // EXPERIMENTAL - SUBJECT TO CHANGE
 	WEBSOCKET_EVENT_PLUGIN_DEACTIVATED  = "plugin_deactivated" // EXPERIMENTAL - SUBJECT TO CHANGE
+	WEBSOCKET_EVENT_LICENSE_CHANGED     = "license_changed"
```

Comment:
Can't see this use in the client (redux/webapp) at the moment but I understand there's more work to do as you've [commented](https://mattermost.atlassian.net/browse/MM-8604?focusedCommentId=45180&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-45180) in Jira ticket.